.github/workflows/cuda-gpu-ci.yml

```diff
@@ -40,7 +48,21 @@ jobs:
           conda activate sklearn
           pip install --verbose --no-build-isolation --config-settings editable-verbose=true --editable .
       - name: Run array API tests
+        id: tests
         run: |
           source "${HOME}/conda/etc/profile.d/conda.sh"
           conda activate sklearn
           pytest -k 'array_api'
+      - name: Create or update PR comment and status
```

Comment:
If we wanted to be very fancy we could add a step at the start of the workflow that post a comment and sets the commit status to "pending" when the workflow starts. And then at the end of the workflow we run this step here.

I think for now it isn't worth the effort/complexity. Seeing the comment when the workflow ends (successful or not) is probably enough.